pkgs/top-level/python-packages.nix

```diff
@@ -4699,6 +4699,23 @@ in modules // {
     };
   };
 
+  pytest-cov = buildPythonPackage rec {
```

Comment:
this package is already in this file but unfortunately without the hyphen: `pytestcov`.